ebu_tt_live/documents/ebutt3_deduplicator-mock.py

```diff
@@ -0,0 +1,42 @@
+from pyxb.binding.basis import NonElementContent, ElementContent
+from pyxb import BIND
+
+style_1 = '<tt:style ebutts:linePadding="0.5c" tts:backgroundColor="rgb(0, 0, 0)" tts:color="rgb(255, 255, 255)" tts:fontFamily="sansSerif" xml:id="SEQ58.defaultStyle1"/>'
+style_2 = '<tt:style ebutts:linePadding="0.5c" tts:backgroundColor="rgb(0, 0, 0)" tts:color="rgb(255, 255, 255)" tts:fontFamily="sansSerif" xml:id="SEQ59.defaultStyle1"/>'
+old_style_list = []
+new_style_list = []
+style_1_list = []
+style_2_list = []
+
+line_padding_begin = style_1.find('linePadding="')
```

Comment:
This seems very brittle. Maybe a regular expression would be better for finding these values.